feat: docs rewrite (#41)
sieve-data/docs: docs/guide/examples/live-audio-transcription.mdx

## **Introduction**

A common usecase for transcription is transcribing RTMP or HLS livestreams. Let's walk through deploying a custom function to Sieve that can ingest a livestream and pass it into other functions using Sieve's function calling capabilities for the purposes of transcription & translation.

## **Clone repo and run the example**
<Steps>
  <Step title="Login with Sieve client">
    To get started, make sure you're logged in via the Sieve Python client:
    ```bash
    sieve login
    ```
  </Step>

  <Step title="Clone example">
    Clone the Sieve examples repository and navigate into the `audio_transcription` directory.
    ```bash
    git clone https://github.com/sieve-community/examples
    cd examples/audio_transcription
    ```

    
  </Step>

  <Step title="Deploy custom function">
    The code we'll be deploying can be found at `examples/audio_transcription/live_transcriber.py`.

>     This is an example ofa custom function to Sieve that can ingest a livestream and pass it into other functions using Sieve's function calling capabilities.

Reviewer: ```suggestion
    This is an example of a custom function to Sieve that can ingest a livestream and pass it into other functions using Sieve's function calling capabilities.
```

>     We won't dive too deep into what the code does but at a high level it:

Reviewer: ```suggestion
    At a high level, this module does the following:
```
